src/test/java/org/isf/lab/test/Tests.java

```diff
@@ -84,6 +100,11 @@
 	@Autowired
 	private ApplicationEventPublisher applicationEventPublisher;
 
+	public Tests(boolean labExtended, boolean labMultipleInsert) {
+		GeneralData.LABEXTENDED = labExtended;
+		GeneralData.LABMULTIPLEINSERT = labMultipleInsert;
```

Comment:
`labMultipleInsert` is defined but never used. Maybe this property is not really needed in "core" component (but only in "gui" to drive the interface selection).

Moreover I found that LabManager needs some code review and optimization that makes again LABMULTIPLEINSERT  quite useless in the "core" component (saving a list or saving a single element list should be the same operation)

Reply:
Removed `LABMULTIPLEINSERT` and added a note that it is only used in the GUI.

I will leave the code review in `LabManager` to those that know much more than I.

Reply:
Thanks!